packages/ods-react/src/components/clipboard/src/components/clipboard-trigger/ClipboardTrigger.tsx

```diff
@@ -55,7 +55,7 @@ const ClipboardTrigger: FC<ClipboardTriggerProp> = forwardRef(({
               name={ ICON_NAME.fileCopy } />
           </Button>
         </TooltipTrigger>
-        { !disabled && <TooltipContent className={ style[ 'clipboard__tooltip' ] }>
+        { !disabled && <TooltipContent className={ classNames(style[ 'clipboard__tooltip-success' ], className) }>
```

Comment:
`clipboard__tooltip-success` ?
if both are actually the same, name both `clipboard__tooltip`